.github/workflows/ci.yml

```diff
@@ -20,26 +20,25 @@ jobs:
           apps: sbt
           jvm: temurin:${{ matrix.jvm }}
       - if: matrix.jvm == '8' || matrix.jvm == '11' || matrix.jvm == '17'
-        run: rm -rf src/sbt-test/skip-sbt1.4
+        run: rm -rf plugin/src/sbt-test/skip-sbt1.4
```

Comment:
let's see if we can keep the existing hierarchy, just diverting the synthetic root project to a dummy directory so that it ignores `./src/`